Makefile/syntax_test_makefile.mak

```diff
@@ -93,6 +93,18 @@ define $(dir)_print =
 lpr $($(dir)_sources)
 endef
 
+define FOO
+  BAR := 1
+  define BAZ
+# ^^^^^^ string.unquoted.makefile - keyword
+    X := 1
+  endef
```

Comment:
Oof, how annoying!

Reply:
What is annoying? The whole content to be a string?

Reply:
The fact that this is actually allowed by the Make syntax, nothing against this PR mate :)